core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -164,6 +164,25 @@ constexpr Array to_array_potentially_narrowing(const U (&init)[M]) {
   }
   return a;
 }
+
+// NOTE Making a copy even when std::is_same<Array, Kokkos::Array<U, M>>::value
+// is true to reduce code conmplexity.  You may change this if you have a good
```

Comment:
conmplexity? :-)

Reply:
I amended the mixup to the commit that introduces the comment.